python_modules/libraries/dagster-dbt/dagster_dbt/cli/utils.py

```diff
@@ -48,60 +58,75 @@ def _create_command_list(
     return prefix + full_command
 
 
-def execute_cli(
-    executable: str,
-    command: str,
-    flags_dict: Mapping[str, Any],
-    log: Any,
-    warn_error: bool,
+def _parse_line(line: str, json_log_format: bool) -> DbtCliEvent:
```

Comment:
this should just be a method on `DbtCliEvent`, just so we colocate this parsing logic

Reply:
agreeed